flowmachine/flowmachine/core/query.py

```diff
@@ -941,7 +941,7 @@ def index_cols(self):
         try:
             if self.subscriber_identifier in cols:
                 ixen.append(self.subscriber_identifier)
-            else:
+            elif "subscriber" in cols:
```

Comment:
Can we please have an explicit `else` branch (which presumably should raise an error)?